pkg/kubelet/rkt/rkt.go

```diff
@@ -142,6 +142,8 @@ const (
 
 	etcHostsPath      = "/etc/hosts"
 	etcResolvConfPath = "/etc/resolv.conf"
+
+	resDone = "done"
```

Comment:
would it be better to call this `restartDone`?

And would it be better to call the channel `restartChannel` instead of `reschan`?